### ISSUE TITLE
Unable to parse dhcp6 relay-reply packets

### ISSUE THREAD

**Issue by flokli** [secdev/scapy#357]
The `DHCP6` class doesn't seem to support parsing dhcp6 relay-reply packets.

There is no `trid` field inside these kind of packets, thats why some bytes at the beginning get cut off, which messes up later layers.

**Comment by guedou** [comment]
Thanks for reporting this issue. Can you provide us an example (for example a PCAP file) ? That will be really useful to fix it quickly.

**Comment by flokli** [comment]
@guedou I digged deeper into the problem, and found it to be a bug in the excepted payload of a DHCPv6 Relay Message option. Also stumbled over some black magic with `ord()`, which might have worked with Python 2, but is still ugly.

As my project using scappy is in Python 3, I wrote a [PR](https://github.com/phaethon/scapy/pull/116) for phaethon/scapy. Probably it's pretty trivial to apply the changes to secdev/scapy, too, but I'd still like to see the projects getting merged ;-)
